app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -69,76 +102,83 @@ def handle_bad_request_error(err, enrollment)
     case err.response&.[](:body)&.[]('responseMessage')
     when IPP_INCOMPLETE_ERROR_MESSAGE
       # Customer has not been to post office for IPP
+      enrollment_outcomes[:enrollments_in_progress] += 1
     when IPP_EXPIRED_ERROR_MESSAGE
       # Customer's IPP enrollment has expired
       enrollment.update(status: :expired)
+      enrollment_outcomes[:enrollments_expired] += 1
     else
       analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
+        **enrollment_attributes(enrollment),
         reason: 'Request exception',
-        enrollment_id: enrollment.id,
-        enrollment_code: enrollment.enrollment_code,
         exception_class: err.class.to_s,
         exception_message: err.message,
       )
+      enrollment_outcomes[:enrollments_errored] += 1
     end
   end
 
   def handle_standard_error(err, enrollment)
+    enrollment_outcomes[:enrollments_errored] += 1
     analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
+      **enrollment_attributes(enrollment),
       reason: 'Request exception',
-      enrollment_id: enrollment.id,
-      enrollment_code: enrollment.enrollment_code,
       exception_class: err.class.to_s,
       exception_message: err.message,
     )
   end
 
   def handle_response_is_not_a_hash(enrollment)
+    enrollment_outcomes[:enrollments_errored] += 1
     analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
+      **enrollment_attributes(enrollment),
       reason: 'Bad response structure',
-      enrollment_id: enrollment.id,
-      enrollment_code: enrollment.enrollment_code,
     )
   end
 
   def handle_unsupported_status(enrollment, status)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_failure(
+    enrollment_outcomes[:enrollments_errored] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
```

Comment:
This is a change to the behavior of the polling job. Previously we would log a message saying this enrollment has failed, but we wouldn't actually update the status of the enrollment. It would be polled again next time.

Now we log an exception message instead of a failure message, and we still don't update the status of the enrollment